src/components/UI/ButtonStyled/DeleteButton.styled.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export const DeleteButton = styled.button`
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	align-self: center;
+	color: white;
+	font-weight: bold;
+	width: 100px;
+	height: 3.2rem;
+	padding: 5px;
+	border: none;
+	position: relative;
+	margin-top: 20px;
+	background: linear-gradient(
+		45deg,
+		rgba(59, 173, 227, 1) 0%,
+		rgba(87, 111, 230, 1) 25%,
+		rgba(152, 68, 183, 1) 51%,
+		rgba(255, 53, 127, 1) 100%
+	);
```

Comment:
Your Editbutton and Deletebutton are almost identical. You should use a single styled component here instead which has a variant prop. WIth this prop you can adapt for minor difference between the buttons.

Reply:
Fixed